### ISSUE TITLE
Can't see my image in motioneyeOS camera

### ISSUE THREAD

**Issue by bend94** [ccrisan/motioneyeos#2186]
Hi

I installed motioneye os on pi zero w.
I boot, i take my brower and i am able to reach MotionEye HTTP interface.
My cam is directly connected on the pi zero w
I can see my cam appearings in motioneye os but without any image : there is a camera with a circle

i got these messages in /var/log/motioneye.log

2019-11-06 20:10:48: [motioneye]  WARNING: Connect error on fd 20: ECONNREFUSED
2019-11-06 20:10:48: [motioneye]  WARNING: Connect error on fd 20: ECONNREFUSED

please advise
thanks

**Comment by mdPlusPlus** [comment]
Which camera are you using?

**Comment by bend94** [comment]
Hi

i use a pi cam, i suspected my cam to be out of order
i ve just ordered a brand new on Amazon
i will see

thanks

**Comment by mdPlusPlus** [comment]
Are you talking about the officical Raspberry Pi Camera Module (v2.1)?

**Comment by bend94** [comment]
Yes the official camera connected on the dedicated cam connector

Le lun. 11 nov. 2019 à 13:33, mdPlusPlus <notifications@github.com> a
écrit :

**Comment by mdPlusPlus** [comment]
Well, this seems unlikely to be honest. The Raspberry Pi Zero (W) has a smaller connector than that camera module. And while there are adapters for "zero cams" to the big connectors of regular Pis, I don't know about any adapter doing the opposite, so from big regular cam to small Zero connector.

Are you sure you're not confusing something here?

**Comment by jasaw** [comment]
Which motioneyeos version are you running? Which web browsers did you use?

**Comment by bend94** [comment]
Amazon has just informed me that i received this cam in my mailbox

Unistorm Raspberry Pi Zero Camera Fisheye Grand Angle 5 MP 1080p Vision
Nocturne Module de caméra pour Raspberry Pi Zero W
<https://www.amazon.fr/gp/product/B074DY544H/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1>



Le lun. 11 nov. 2019 à 13:33, mdPlusPlus <notifications@github.com> a
écrit :

**Comment by hexordecimal** [comment]
did you enable the camera with raspi-config? thats what i think when i see connection refused. Also at one point i think i had a problem with the camera not appearing because of my resolution settings.
